content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -1037,6 +1038,12 @@ Each feature gate is designed for enabling/disabling a specific feature:
   topology of the cluster. See
   [ServiceTopology](/docs/concepts/services-networking/service-topology/)
   for more details.
+- `ServiceIPStaticSubrange`: Enable a new strategy for Services ClusterIP allocations, so users can
+  subdivide the ClusterIP range. Dynamic allocated ClusterIP addresses will be allocated preferently
+  from the upper range allowing users to assign static ClusterIPs from the lower range with a low
+  risk of collision. See
+  [ServiceIPStaticSubrange](https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/3070-reserved-service-ip-range#proposed-services-clusterips-allocation-model)
+  for more details.
```

Comment:
We don't ask readers to refer to KEPs to understand Kubernetes. Please instead add a document that describes the feature into the website repo.

If you're looking for a place to hang that new documentation off, it may help to move PR https://github.com/kubernetes/website/pull/30817 forward first - but you don't have to.

Reply:
Yeah, we should do #30817 , but I'll not mix it with this, the scope for #30817 is too broad and there are certain concepts that doesn't apply anymore, I'll comment there